lib/monitoring/addon/components/nodes-live/component.js

```diff
@@ -230,12 +188,12 @@ export default NodesReservation.extend(Metrics, {
   }),
 
   meomoryLiveTitle: computed('nodes.@each.{memory}', 'memoryLive', 'intl.locale', function() {
-    const { memoryLive, nodes = [] } = this
+    const { graphs, nodes = [] } = this
 
-    if (!memoryLive) {
+    if (!graphs) {
       return 'loading'
     }
-    const { nodeCapcity = {}, graphs = [] } = this
+    const { nodeCapcity = {} } = this
```

Comment:
Is there any reason this can't be included on the destructuring assignment on line 191?